src/components/MessagesList/MessagesGroup/Message/MessagePart/Forwarder.vue

```diff
@@ -188,12 +192,11 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-
 .forwarded-confirmation {
 	&__emptycontent {
-		width: 280px;
+		width: 100%;
 		text-align: center;
-		margin: 20px !important;
+		padding-top: 20px !important;
```

Comment:
This still needs fixing.

```suggestion
		margin-top: 15vh !important;
```

Looks best I think:

Before | After
---|---
![Bildschirmfoto von 2022-03-14 12-16-38](https://user-images.githubusercontent.com/213943/158161861-6bb87831-3ae4-4097-a6c9-74be60b3ec8b.png) | ![Bildschirmfoto von 2022-03-14 12-18-21](https://user-images.githubusercontent.com/213943/158161868-ab800ce4-3d61-4b1a-9d1a-918e955d3a9b.png)

Reply:
Ok :)